packages/dev/core/src/Misc/fileTools.ts

```diff
@@ -216,9 +220,23 @@ export const LoadImage = (
     const img = new Image();
     SetCorsBehavior(url, img);
 
+    let handlersList: { target: any; name: string; handler: any }[] = [];
+
+    const loadHandlersList = () => {
+        handlersList.forEach((handler) => {
+            handler.target.addEventListener(handler.name, handler.handler);
+        });
+    };
+
+    const unloadHandlersList = () => {
+        handlersList.forEach((handler) => {
+            handler.target.removeEventListener(handler.name, handler.handler);
+        });
+        handlersList = [];

```

Comment:
```suggestion
        handlersList.length = 0;
```

Avoid garbage collection